### PR TITLE
feat: User model

### PR DESCRIPTION
**Pull request by Juliocho990**
## Summary

* User model was added.
* User model migration was implemented.

## Known Issues

None

## Trello Card

https://trello.com/c/GGbHY4Ca/21-modelo-de-usuario

### CODE REVIEW

File: app/models/user.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
+module.exports = (sequelize, DataTypes) => {
+  const User = sequelize.define(
+    'users',
```
Comment (blueJayWays) [inline app/models/user.js]: for convention, model's name must be singular.

File: migrations/migrations/20210308144235-user-model.js [hunk 2]
```diff
@@ -0,0 +1,38 @@
+module.exports = {
+  up: (queryInterface, Sequelize) =>
+    queryInterface.createTable('users', {
```
Comment (blueJayWays) [inline migrations/migrations/20210308144235-user-model.js]: bdd table name must be singular. For convention.

File: app/models/user.js [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+        validate: {
+          isEmail: true
+        }
+      },
+      password: {
+        type: DataTypes.STRING,
+        allowNull: false
+      }
+    },
+    {
```
Comment (blueJayWays) [inline app/models/user.js:26]: maybe you need a `timestamps: true`. It's useful to know when a record was created or updated, and maybe in the future you will need  it.

Comment (Juliocho990) [reply app/models/user.js:26]: ok

File: migrations/migrations/20210308144235-user-model.js (hunk 2 above)
Comment (Juliocho990) [reply migrations/migrations/20210308144235-user-model.js]: ok

File: app/models/user.js (hunk 1 above)
Comment (Juliocho990) [reply app/models/user.js]: ok

**Review by blueJayWays — APPROVED** [review]

**Pull request merged**